CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+Contributing
+============
+
+-   [Fork](https://help.github.com/articles/fork-a-repo) the [notifier on github](https://github.com/bugsnag/bugsnag-laravel)
+-   Build and test your changes. Run the tests using [phpunit](https://phpunit.de) (vendored to `vendor/bin/phpunit`)
```

Comment:
If we’re using the makefile, then add instructions here for how to build and test.